Makefile

```diff
@@ -371,13 +371,13 @@ CHECK		= sparse
 
 # Use the wrapper for the compiler.  This wrapper scans for new
 # warnings and causes the build to stop upon encountering them.
-CC		= $(srctree)/scripts/gcc-wrapper.py $(REAL_CC)
+# CC		= $(srctree)/scripts/gcc-wrapper.py $(REAL_CC)
```

Comment:
This and the REAL_CC change disable this nasty python wrapper for GCC that stops build on any warning.